Modules/_xxsubinterpretersmodule.c

```diff
@@ -627,6 +627,36 @@ _channelends_associate(_channelends *ends, int64_t interp, int send)
     return 0;
 }
 
+static int64_t *
+_channelends_list_interpreters(_channelends *ends, int64_t *count, int send)
+{
+    int64_t *ids = NULL;
+    int64_t numopen;
+
+    assert(ends != NULL);
+    assert(count != NULL);
```

Comment:
Considering that this is an internal function and we control all call sites, these assertions are unnecessary.